src/pages/ViewInterviewPage.tsx

```diff
@@ -54,25 +62,25 @@ const InterviewView = ({ interview }: { interview: HydratedInterview }) => {
   } = interview;
 
   const statusColors = {
-    active: 'green',
-    pending: 'yellow',
-    inactive: 'gray',
+    active: "green",
+    pending: "yellow",
+    inactive: "gray",
   };
 
   const startDateStr = new Date(dateEffective).toLocaleDateString(undefined, {
-    weekday: 'long',
-    year: 'numeric',
-    month: 'long',
-    day: 'numeric',
+    weekday: "long",
+    year: "numeric",
+    month: "long",
+    day: "numeric",
   });
 
   const endDate = new Date(dateEffective);
   endDate.setDate(endDate.getDate() + 7);
   const endDateStr = endDate.toLocaleDateString(undefined, {
-    weekday: 'long',
-    year: 'numeric',
-    month: 'long',
-    day: 'numeric',
+    weekday: "long",
```

Comment:
I think you should use the formatter that we have included with `npm run lint` before committing so we are consistent across all and it doesn't have edits like this in all your files.